configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
```

Comment:
Yes, given that types are architecturally necessary, we will proceed to implementing the model where types are explicitly listed by the operator as stand-alone concepts. Once we have it working, then we will figure out the appropriate syntax to express type inference.

I think we can improve the name of entities in the example to improve the situation without having to change anything else.

Once again, I believe there is an upper limit to the conceptual complexity and total size that is reasonable for people to author against in YAML. We will need a bona fide UI to make all this usable.